app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/DataCreationActivity.kt

```diff
@@ -1,13 +1,14 @@
 package com.github.HumanLearning2021.HumanLearningApp
 
+import android.content.Intent
 import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.widget.EditText
-import android.widget.Toast
 import androidx.appcompat.app.AppCompatActivity
-
 import com.github.HumanLearning2021.HumanLearningApp.databinding.ActivityDataCreationBinding
+import com.github.HumanLearning2021.HumanLearningApp.model.DummyCategory
+
 
 class DataCreationActivity : AppCompatActivity() {
```

Comment:
```suggestion
class DatasetCreationActivity : AppCompatActivity() {
```
it's the creation of an data**set** no?